--- before
+++ after
@@ -103,4 +103,4 @@
 smtpd_tls_loglevel = 1
 tls_server_sni_maps = 
-smtpd_recipient_restrictions = permit_mynetworks, permit_sasl_authenticated, reject_unauth_destination, reject_unknown_sender_domain, check_policy_service unix:private/policyd-spf, reject_rbl_client dnsbl.set1-lab.test
+smtpd_recipient_restrictions = permit_mynetworks, permit_sasl_authenticated, reject_unauth_destination, reject_unknown_sender_domain, check_policy_service unix:private/policyd-spf
 smtpd_client_message_rate_limit = 30
